torch/fx/node.py

```diff
@@ -25,8 +25,8 @@ def __init__(self, graph: 'Graph', name: str, op: str, target: Target,
                  args: Tuple[Argument, ...], kwargs: Dict[str, Argument]) -> None:
         self.graph = graph
         self.name = name  # unique name of value being created
-        assert op in ['placeholder', 'call_method', 'call_module', 'call_function', 'get_attr']
-        self.op = op  # the kind of operation = placeholder|call_method|call_module|call_function|get_attr
+        assert op in ['placeholder', 'call_method', 'call_module', 'call_function', 'get_attr', 'return']
```

Comment:
the node that creates the "return" node is called ".output"  that seems inconsistent. Rename return to output?